{
  log.info("Starting testCreateWithBPCanceledFromUserApi");
  try {
    testCreateAndRecreate(true);
    assertListenerStatus();
  }
 catch (  EntitlementUserApiException e) {
    log.error("Unexpected exception",e);
    Assert.fail(e.getMessage());
  }
}
